expenses/models.py

```diff
@@ -24,15 +24,19 @@ class RequisitionStatus(models.Model):
     PENDING_CHANGES = "Pending Changes"
     READY_TO_ORDER = "Ready to Order"
     ORDERED = "Ordered"
+    PARTLY_RECEIVED = "Partly Received"
```

Comment:
I think "partially received" might be slightly better to use for this

Reply:
Fixed